recipes/libcurl/all/conanfile.py

```diff
@@ -17,46 +17,48 @@ class LibcurlConan(ConanFile):
     generators = "cmake", "cmake_find_package_multi", "pkg_config"
 
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False],
-               "fPIC": [True, False],
-               "with_ssl": [False, "openssl", "wolfssl", "schannel", "darwinssl"],
-               "with_openssl": [True, False, "deprecated"],
-               "with_wolfssl": [True, False, "deprecated"],
-               "with_winssl": [True, False, "deprecated"],
-               "darwin_ssl": [True, False, "deprecated"],
-               "with_ldap": [True, False],
-               "with_libssh2": [True, False],
-               "with_libidn": [True, False],
-               "with_librtmp": [True, False],
-               "with_libmetalink": [True, False],
-               "with_libpsl": [True, False],
-               "with_largemaxwritesize": [True, False],
-               "with_nghttp2": [True, False],
-               "with_zlib": [True, False],
-               "with_brotli": [True, False],
-               "with_zstd": [True, False],
-               "with_c_ares": [True, False],
-               }
-    default_options = {"shared": False,
-                       "fPIC": True,
-                       "with_ssl": "openssl",
-                       "with_openssl": "deprecated",
-                       "with_wolfssl": "deprecated",
-                       "with_winssl": "deprecated",
-                       "darwin_ssl": "deprecated",
-                       "with_ldap": False,
-                       "with_libssh2": False,
-                       "with_libidn": False,
-                       "with_librtmp": False,
-                       "with_libmetalink": False,
-                       "with_libpsl": False,
-                       "with_largemaxwritesize": False,
-                       "with_nghttp2": False,
-                       "with_zlib": True,
-                       "with_brotli": False,
-                       "with_zstd": False,
-                       "with_c_ares": False,
-                       }
+    options = {
+        "shared": [True, False],
```

Comment:
```suggestion
       "shared": [True, False],
```

If we already re-formatting then let's be nit-picking here, this line has a space more than the rest

Reply:
Actually, that was the only line with the intended indentation (=multiple of 4).
Thanks for the thorough review :stuck_out_tongue:

Reply:
Haha, sorry, correct indentation is hard to see in the web UI